vale-linting/action.yml

```diff
@@ -21,5 +21,3 @@ runs:
         exclude: ${{ inputs.exclude }}
```

Comment:
(for line 19).
There is no input called `files`. [action.yaml](https://github.com/extenda/vale-action/blob/master/action.yml). It needs to be removed from composite action or added to our fork.

The error:
```
Warning: Unexpected input(s) 'files', valid inputs are ['entryPoint', 'args', 'styles', 'config', 'exclude', 'onlyAnnotateModifiedLines', 'passCondition', 'debug']
```